files/pt-br/learn/css/first_steps/how_css_works/index.md

```diff
@@ -51,7 +51,7 @@ Quando um navegador redenriza um documento, ele combina o documento com suas inf
 
 O diagrama a seguir também apresenta uma visão simples do processo.
 
-![](https://mdn.mozillademos.org/files/11781/rendering.svg)
+![Rendering process overview](https://github.com/mdn/content/blob/main/files/en-us/learn/css/first_steps/how_css_works/rendering.svg)
```

Comment:
Hi @BiancaPereira, we can just use `rendering.svg` to refer the image file here, mdn/yari will fallback to find the image in en-US.

```suggestion
![Rendering process overview](rendering.svg)
```

Reply:
Thanks @yin1999!